AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -315,6 +315,7 @@ - (void)_initializeInstance
   _environmentState = ASEnvironmentStateMakeDefault();
   
   _calculatedDisplayNodeLayout = std::make_shared<ASDisplayNodeLayout>();
```

Comment:
Might be worth considering some macro to access the common internals like -> layout, safely returning nil if the struct is nullptr.  Not needed now but it's tough to judge the correctness of the other code in the file with only one of the layout variables being non-nullptr.